src/webui/www/private/views/rssDownloader.html

```diff
@@ -535,11 +535,12 @@
         };
 
         const removeRules = (rules) => {
+            let encodeReules = rules.map(function(rule) { return encodeURIComponent(rule); });
```

Comment:
Use `const` and fix typo "Reules" -> "Rules".
Also I would name it `encodedRules`

Reply:
Fixed.